### PR TITLE
Add an option to use the SSL integration between LDAP and Kafka

### PR DESCRIPTION
**Pull request by purbon**
This PR adds an option to use the SSL integration between LDAP and Kafka. 

If you start the playbook with ```./up``` the default, no secured configuration is used, while if you start with ```./up --ssl``` the TLS connection is put in place. This is useful to show the different ways of working from a single playbook.

Looking forward to know your thoughts.

@Dabz @sknop

### CODE REVIEW

File: ldap/kafka/server-with-ssl.properties [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+sasl.mechanism.inter.broker.protocol=SCRAM-SHA-256
+security.inter.broker.protocol=SASL_PLAINTEXT
+allow.everyone.if.no.acl.found=false
+super.users=User:kafka
+authorizer.class.name=kafka.security.auth.SimpleAclAuthorizer
+
+# Configure authorizer
+authorizer.class.name=io.confluent.kafka.security.ldap.authorizer.LdapAuthorizer
+# LDAP provider URL
+ldap.authorizer.java.naming.provider.url=ldap://ldap:636/DC=CONFLUENT,DC=IO
```
Comment (Dabz) [inline ldap/kafka/server-with-ssl.properties]: quick and stupid question, shouldn't it be ldaps:// instead of ldap:// ?

**Comment by Dabz** [comment]
I tested it locally, it seems to be working nicely. I am just surprised as there is a `ldap.authorizer.java.naming.provider.url=ldap://ldap:636/`, I would be expecting to rely on `ldaps://` instead of the ldap syntax no? Anyway, it seems to be working in both cases (with `ldap` and `ldaps`). What's your opinion on `ldap://` vs `ldaps://`? 
Otherwise, it should be ready for merging ;)

**Comment by purbon** [comment]
I like to use ldaps, is a very fair point. Updating the PR right now.

**Comment by purbon** [comment]
@Dabz done! happy to contribute this.

**Pull request merged**